src/index.ts

```diff
@@ -26,58 +23,51 @@ type CLIParams = {
   };
 };
 
-const enrichTrace = async (transactionName: string, transactionType: string, service: string, api: ApmClient, params: Omit<CLIParams['param'], 'transactionType'>) => {
-  const traceSamples = await api.getTraceSamples({
-    ...params,
-    transactionName,
-    transactionType,
-    service,
-  });
-
-  const uniqueTraceSamples = _.uniqBy(traceSamples, 'traceId')
-
-  const traces = await bluebird.map<_, Events[]>(uniqueTraceSamples, ({ traceId }) => api.getTrace({
-    id: traceId,
-    start: params.start,
-    end: params.end
-  }), { concurrency: 10 });
-
-
-  const rawTraces = traces.map(events => tree(events.map(sanitize.scalability))).filter(e => !_.isEmpty(e))
-
-  return {
-    service,
-    transactionType,
-    transactionName,
-    traces: rawTraces
-  }
-};
-
 const apmParser = async ({ param, client }: CLIParams) => {
```

Comment:
Nit: Let's call something like scalability for a while or extract the logic to scalability function since we are gonna use this for esperf also

Reply:
Agreed to leave as is